BP/common/landed_titles/BP_e_arabia.txt

```diff
@@ -10,17 +10,17 @@ e_arabia = {
 	
 	ai_primary_priority = {
 		if = {
-			limit = { culture_group = culture_group:central_semitic_group }
+			limit = { always = yes }
 			add = @correct_culture_primary_score
 		}
 	}
 	
 	cultural_names = {
-		aramean = cn_urbi
-		ashkenazi = cn_aravah
-		sephardi = cn_aravah
-		hebrew = cn_aravah
-		greek = cn_aravia
+		#aramean = cn_urbi
+		#ashkenazi = cn_aravah
+		#sephardi = cn_aravah
+		#hebrew = cn_aravah
+		#greek = cn_aravia
```

Comment:
Why are these names being commented out?

Reply:
Caused an suite of errors because of changes from cultures to name_lists; patched in PR #75 (which merged @Symmachus additional work for culture place names).